### PR TITLE
Updates links to policies, adds a new point to test the language pack

### PR DESCRIPTION
**Pull request by Crizz0**
Please check the new wording, thank you! @phpbb/native 

@phpbb/extensions-team Feel free to review it.

### CODE REVIEW

File: language/en/types/translation.php [hunk 1]
```diff
@@ -70,8 +70,9 @@
...
 [list=1][*]Make the necessary changes to correct any problems (listed below) that resulted in your language pack being denied.
-[*]Please ensure your language pack is up-to-date with the latest version of phpBB (see the [url=http://www.phpbb.com/downloads/]Downloads[/url] page).
-[*]Please ensure that you comply with our [url=http://www.phpbb.com/community/viewtopic.php?f=79&t=2117453]Important Read Me![/url] and our [url=http://www.phpbb.com/community/viewtopic.php?f=79&t=2125191]Language Packs Submission Policy[/url].
+[*]Please ensure your language pack is up-to-date with the latest version of phpBB (see the [url=https://www.phpbb.com/customise/db/translation/british_english/]Downloads[/url] page).
+[*]Please ensure that you comply with our [url=https://www.phpbb.com/community/viewtopic.php?f=79&t=2117453]Important Read Me![/url] (where you\'ll find the latest submission policies).
+[*]Please ensure that your language pack is working correctly. Therefor please test it elaborately with a suitable phpBB-installation. Our [url=https://www.phpbb.com/community/viewtopic.php?f=79&t=2205631]Translation Validator Tool[/url] will help you finding bugs on your own.
```
Comment (DavidIQ) [inline language/en/types/translation.php]: Some corrections:
> Please ensure that your language pack is working correctly. ~~Therefor~~ **Therefore** please test it elaborately with a suitable ~~phpBB-installation~~ **phpBB installation**. Our [url=https://www.phpbb.com/community/viewtopic.php?f=79&t=2205631]Translation Validator Tool[/url] will help you ~~finding~~ **find** bugs on your own.

File: language/en/types/translation.php [hunk 2]
```diff
@@ -70,8 +70,9 @@
 
 To correct the problem(s) with your language pack, please following the below instructions:
 [list=1][*]Make the necessary changes to correct any problems (listed below) that resulted in your language pack being denied.
-[*]Please ensure your language pack is up-to-date with the latest version of phpBB (see the [url=http://www.phpbb.com/downloads/]Downloads[/url] page).
-[*]Please ensure that you comply with our [url=http://www.phpbb.com/community/viewtopic.php?f=79&t=2117453]Important Read Me![/url] and our [url=http://www.phpbb.com/community/viewtopic.php?f=79&t=2125191]Language Packs Submission Policy[/url].
+[*]Please ensure your language pack is up-to-date with the latest version of phpBB (see the [url=https://www.phpbb.com/customise/db/translation/british_english/]Downloads[/url] page).
+[*]Please ensure that you comply with our [url=https://www.phpbb.com/community/viewtopic.php?f=79&t=2117453]Important Read Me![/url] (where you\'ll find the latest submission policies).
```
Comment (vinny) [inline language/en/types/translation.php]: `you’ll`

File: language/en/types/translation.php [hunk 3]
```diff
@@ -70,8 +70,9 @@
 
 To correct the problem(s) with your language pack, please following the below instructions:
 [list=1][*]Make the necessary changes to correct any problems (listed below) that resulted in your language pack being denied.
-[*]Please ensure your language pack is up-to-date with the latest version of phpBB (see the [url=http://www.phpbb.com/downloads/]Downloads[/url] page).
-[*]Please ensure that you comply with our [url=http://www.phpbb.com/community/viewtopic.php?f=79&t=2117453]Important Read Me![/url] and our [url=http://www.phpbb.com/community/viewtopic.php?f=79&t=2125191]Language Packs Submission Policy[/url].
+[*]Please ensure your language pack is up-to-date with the latest version of phpBB (see the [url=https://www.phpbb.com/customise/db/translation/british_english/]Downloads[/url] page).
+[*]Please ensure that you\'ll comply with our [url=https://www.phpbb.com/community/viewtopic.php?f=79&t=2117453]Important Read Me![/url] (where you\'ll find the latest submission policies).
```
Comment (DavidIQ) [inline language/en/types/translation.php]: Let's reword this to:
> Please ensure that you are in compliance with the [url=https://www.phpbb.com/community/viewtopic.php?f=79&t=2117453]latest submission policies[/url].

We should eventually move those submission policies to somewhere on the main site instead of the forum.
